tests/RandomTest.php

```diff
@@ -399,7 +398,10 @@ public function testGetTitle()
         $m->title_field = 'my_name';
         $m->load(2);
         $this->assertEquals(2, $m->getTitle()); // loaded returns id value
-        $this->assertEquals([1 => 1, 2 => 2], $m->getTitles()); // all titles (my_name)
+
+        $this->expectException(Exception::class);
```

Comment:
No. `getTitles()` should work and if it needs duplicate, then that should be done in getTitles() or export() methods.

Reply:
![image](https://user-images.githubusercontent.com/2228672/115303309-c1314d00-a163-11eb-8bcd-4615fc5c5941.png)